bpf-gpl/parsing.h

```diff
@@ -97,9 +97,104 @@ static CALI_BPF_INLINE int parse_packet_ip(struct cali_tc_ctx *ctx) {
 	return 0;
 
 allow_no_fib:
-	return -1;
+	return -2;
```

Comment:
As also noted below, can we #define constants for these -1 and -2 numbers?

Reply:
Yes, sure. I wanted to do that, but noticed in other functions return codes are used like this. But it would be much cleaner with constants. Will fix it.